src/components/Composer/index.native.js

```diff
@@ -77,6 +112,7 @@ class Composer extends React.Component {
         if (!this.props.forwardedRef || !_.isFunction(this.props.forwardedRef)) {
             return;
         }
+        this.textInput.setTextAndSelection = this.setTextAndSelection;
```

Comment:
Hey, it's "e're adding it to the ref so that other components can use it". However, i just got rid of that whole `viewCommands` stuff. So i think its fine to keep it (i added a comment to make it more clear why its there)